posts/2021-03-24-microprofile40-open-liberty-21003.adoc

```diff
@@ -0,0 +1,442 @@
+. link:https://github.com/eclipse/microprofile-opentracing/releases/tag/2.0[MicroProfile OpenTracing 2.0]
+. link:https://github.com/eclipse/microprofile-rest-client/releases/tag/2.0[MicroProfile Rest Client 2.0]
+
+In the next section, we will go through each MicroProfile Specification and take a further look at their new and exciting features.
+
+== Getting started
+
+MicroProfile 4.0 implementation is new in Open Liberty 21.0.0.3 - you can use any version from 21.0.0.3 and onwards.
+
+First, you need to add the following link:{url-prefix}/guides/maven-intro.html[Maven] dependency
+to your pom.xml:
+
+[source,xml]
+----
+
+<dependency>
+    <groupId>org.eclipse.microprofile</groupId>
+    <artifactId>microprofile</artifactId>
+    <version>4.0.1</version>
+    <type>pom</type>
+    <scope>provided</scope>
+</dependency>
+
+----
+
+or, if you use link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+[source,gradle]
+----
+dependencies {
+    mpRestClient group: 'org.eclipse.microprofile', name: 'microprofile', version: '4.0.1'
+}
+----
+Please note: you need to use the version `4.0.1` as the version `4.0` does not pull in any dependencies by mistake (see link:https://github.com/eclipse/microprofile/issues/205[here] for more details).
+
+Also, make sure to configure your Liberty server with the `microProfile-4.0` feature in the server.xml:
+[source,xml]
+----
+<server>
+  <featureManager>
+    <feature>microProfile-4.0</feature>
+    <!-- ... -->
+  </featureManager>
+  <!-- ... -->
+</server>
+----
+
+
+That's it! Now that we've got our development and deployment environments set up, it's time to dive into each individual MicroProfile Specification to learn the amazing updates!
+
+. <<config-2.0, What's new in MicroProfile Config 2.0>>
+. <<fault-tolerance-3.0, New features in MicroProfile Fault Tolerance 3.0>>
+. <<health-3.0, New changes in MicroProfile Health 3.0>>
+. <<metrics-3.0, What's New in MicroProfile Metrics 3.0>>
+. <<open-tracing-2.0, New updates in  MicroProfile Open Tracing 2.0>>
+. <<open-api-2.0, What's New in MicroProfile Open API 2.0>>
+. <<rest-client-2.0, Awesome features in MicroProfile Rest Client 2.0>>
+. <<jwt-1.2, Must known features in MicroProfile JWT Auth 1.2>>
+
+
+
+[#config-2.0]
+== What's New in MicroProfile Config 2.0
+
+Quite a few nice capabilities were provided by MicroProfile Config 2.0, such as Config Profile, Config Properties, ConfigValue support, and more.
+
+Open Liberty's MicroProfile Config 2.0 feature is based on link:https://github.com/smallrye/smallrye-config[SmallRye's implementation] of the specification.
+
+Watch this space for a new blog post dedicated to MicroProfile Config 2.0.
+
+[#fault-tolerance-3.0]
+== What's New in MicroProfile Fault Tolerance 3.0
+
+MicroProfile Fault Tolerance allows you to easily apply strategies for mitigating failure to their code. It provides annotations which you can add to methods to use bulkhead, circuit breaker, retry, timeout and fallback strategies.
```

Comment:
First sentence references you and then their, this is confusing. Is it for mitigating failures to your code? Should their actually be your?